gdx/src/com/badlogic/gdx/maps/tiled/BaseTmxMapLoader.java

```diff
@@ -88,6 +86,16 @@ public Array<AssetDescriptor> getDependencies (String fileName, FileHandle tmxFi
 		return getDependencyAssetDescriptors(tmxFile, textureParameter);
 	}
 
+	/** Gets a map of the object ids to the {@link MapObject} instances. The returned map is a copy of the internal map. Returns
+	 * null if {@link #loadTiledMap(FileHandle, Parameters, ImageResolver)} has not been called yet. The internal map is reset when
+	 * the load tiled map method is called.
+	 *
+	 * @return the map of the object ids to the {@link MapObject} instances, or null if the load tiled map method has not been
+	 *         called yet. */
+	public IntMap<MapObject> getIdToObject () {
```

Comment:
We have a `@Null` annotation for nullable return values

Reply:
Added the `@Null tag`